src/main/java/group2/cms/service/AuthorService.java

```diff
@@ -23,47 +27,66 @@
     @Autowired
     private AuthorRepository authorRepository;
 
-    public Author addAuthor(Long userID, Long paperID){
+    public AuthorDTO addAuthor(AuthorDTO authorDTO){
+        var userID = authorDTO.getUserID();
```

Comment:
I think that it is better to do the conversion before any other call and use the object that you got from the conversion

Reply:
The DTO should be used only to get the data from the front, send it back and nothing more